definition/IRoom.ts

```diff
@@ -160,6 +160,26 @@ export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | '
 	ts: Date;
 	label?: string;
 	crmData?: unknown;
+
+	// optional keys for closed rooms
+	closer?: 'user' | 'visitor';
+	closedBy?: {
+		_id: string;
+		username: IUser['username'];
+	};
+	serviceTimeDuration?: string;
```

Comment:
There's already a serviceTimeDuration inside metrics, is this one also necessary?